mteb/tasks/Retrieval/eng/LEMBNarrativeQARetrieval.py

```diff
@@ -0,0 +1,67 @@
+import datasets
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskRetrieval import AbsTaskRetrieval
+
+
+class LEMBNarrativeQARetrieval(AbsTaskRetrieval):
+    _EVAL_SPLIT = "test"
+
+    metadata = TaskMetadata(
+        name="LEMBNarrativeQARetrieval",
+        dataset={
+            "path": "dwzhu/LongEmbed",
+            "revision": "6e346642246bfb4928c560ee08640dc84d074e8c",
+            "name": "narrativeqa",
+        },
+        reference="https://huggingface.co/datasets/dwzhu/LongEmbed",
+        description=("narrativeqa subset of dwzhu/LongEmbed dataset."),
+        type="Retrieval",
+        category="s2p",
+        eval_splits=[_EVAL_SPLIT],
+        eval_langs=["eng-Latn"],
+        main_score="ndcg_at_10",
+        date=None,
+        form=["written"],
+        domains=["Fiction", "Non-fiction"],
+        task_subtypes=["Article retrieval"],
+        license=None,
+        socioeconomic_status=None,
```

Comment:
The socioeconomic status of the text creators (lawyers is high, social media is mixed, news articles would be medium, but depends on the outlet)

It is just a rough estimate

Reply:
ok, I see now. So the document here is referring to the text creators, rather than the data itself, right? https://github.com/embeddings-benchmark/mteb/blob/main/mteb/abstasks/TaskMetadata.py#L139

Reply:
Ahh right, yea that is not very clear. Feel free to suggest a reformulation and add a point to (bug)fixes for yourself.